core/bones/json.py

```diff
@@ -12,10 +13,13 @@ class JsonBone(RawBone):
 
     type = "raw.json"
 
-    def __init__(self, indexed: bool = False, multiple: bool = False, languages: bool = None, *args, **kwargs):
+    def __init__(self, indexed: bool = False, multiple: bool = False, languages: bool = None, schema: object = None,
+                 *args,
+                 **kwargs):
         assert not multiple
         assert not languages
         assert not indexed
+        self.schema = schema
```

Comment:
Let's validate the schema on bone creation.
```suggestion
        # Validate the schema, if it's invalid a SchemaError will be raised
        jsonschema.validators.validator_for(False).check_schema(schema)
        self.schema = schema
```

Reply:
@ArneGudermann we should wrap this with an `if`. I get 
```py
jsonschema.exceptions.SchemaError: None is not of type 'object', 'boolean'
```